lib/LicenseTermsList/LicenseTermsList.js

```diff
@@ -19,9 +19,18 @@ export default class LicenseTermsList extends React.Component {
       label: PropTypes.string,
       name: PropTypes.string,
       primary: PropTypes.bool,
+      defaultInternal: PropTypes.bool,
     })),
   };
 
+  renderVisibility = (internal, term) => {
+    if ((internal === false) || (!internal && term.defaultInternal === false)) {
+      return <span data-test-term-visibility={term.name}>{<FormattedMessage id="stripes-erm-components.term.internalFalse" />}</span>;
+    }
+
+    return <span data-test-term-visibility={term.name}>{<FormattedMessage id="stripes-erm-components.term.internalTrue" />}</span>;
```

Comment:
Finally, you can simplify this method by always returning the same span and FormattedMessage but just varying the `id` you pass into the FormattedMessage. Eg,
```
const internalId = ((internal === false) || (internal === undefined && term.defaultInternal === false)) ?
  'internalFalse' : 'internalTrue';

return (
  <span data-test-term-visibility={term.name}>
    <FormattedMessage id={`stripes-erm-components.term.${internalId}`} />
  </span>
);
```